visualizer/src/Controls/ImageControls/RawControls/LayerOptions.js

```diff
@@ -0,0 +1,75 @@
+/** Modified from https://github.com/hms-dbmi/viv */
+import ClickAwayListener from '@material-ui/core/ClickAwayListener';
+import IconButton from '@material-ui/core/IconButton';
+import MenuItem from '@material-ui/core/MenuItem';
+import MenuList from '@material-ui/core/MenuList';
+import Paper from '@material-ui/core/Paper';
+import Popper from '@material-ui/core/Popper';
+import { makeStyles } from '@material-ui/core/styles';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import { useSelector } from '@xstate/react';
+import React, { useReducer, useRef } from 'react';
+import { useRaw } from '../../../ProjectContext';
+import ColorPalette from './ColorPalette';
+
+const useStyles = makeStyles(() => ({
```

Comment:
This CSS was from the viv project?

Reply:
Yes, and there's a `/** Modified from https://github.com/hms-dbmi/viv */` comment at the top of the file to attribute it